Extensions/video_downloader.css

```diff
@@ -0,0 +1,33 @@
+.xvd-button{
+	position: absolute;
+	top: 2px;
+	left: 2px;
+	display: block;
+	overflow: hidden;
+	height: 32px;
+	width: 32px;
+	margin: 0px;
+	padding: 0px;
+	padding-left: 32px;
+	background-image: none !important;
+	z-index: 2;
+	border-radius: 6px;
+	opacity: .5;
+	transition: background-color .2s, opacity .2s;
+}
+.xvd-button: hover, .xvd-button: focus{
+	background-color: rgba(0,0,0,.25);
+	opacity: 1;
+}
+.xvd-button: : before{
```

Comment:
this selector is broken

also you didn't put any spaces before the braces

Reply:
oops, that's what using ctrl+f gets you, lol. thanks for pointing that out